src/sentry/models/transaction_threshold.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+from django.db import models
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class TransactionMetric(Enum):
+    DURATION = 1
+    LCP = 2
+    FCP = 3
+
+
+class TransactionThreshold(Model):
+    __core__ = False
+
+    # max_length here is based on the maximum for transactions in relay
+    transaction = models.CharField(max_length=200, null=True)
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
+    organization = FlexibleForeignKey("sentry.Organization")
+    threshold = models.IntegerField()
+    metric = models.PositiveSmallIntegerField(default=TransactionMetric.DURATION.value)
```

Comment:
can we list all the allowed metrics as `choices=...` here?

Reply:
I anticipate we'll be adding more metrics (for instance when we have a good mobile metric) and we'll need a migration every time we add new metrics if we added choices here. We can just handle the validation in the serializer I think and make sure to raise an error if it's not a valid choice.